docs: initial Diataxis site (#248)
jnsgruk/concierge: docs/explanation/index.md

---
myst:
  html_meta:
    description: Background reading on Concierge — what it is for and why prepare and restore are strict opposites.
---

(explanation)=
# Explanation

> Background and context for how Concierge thinks about the machines it manages.

Reviewer: How about this instead?

```suggestion
Background and context for how Concierge interacts with the machines it manages.
```
